wear/src/main/java/com/android/wear/presentation/MainActivity.kt

```diff
@@ -0,0 +1,56 @@
+package com.android.wear.presentation
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.wear.compose.material.*
+import com.android.library.LocationLabelAndData
+import com.android.wear.presentation.theme.GpstestTheme
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            WearApp(LocationLabelAndData.statusSample)
+        }
+    }
+}
+
+@Composable
+fun WearApp(satStatues: List<String>) {
+    GpstestTheme {
+        val listState = rememberScalingLazyListState()
+        Scaffold(
+            timeText = {
+                if (!listState.isScrollInProgress) {
+                    TimeText()
+                }
+            },
+            positionIndicator = {
+                PositionIndicator(
+                    scalingLazyListState = listState
+                )
+            }
+        ) {
+            val contentModifier = Modifier
+                .fillMaxWidth()
+                .padding(bottom = 8.dp)
+            ScalingLazyColumn(
+                modifier = contentModifier,
+                autoCentering = AutoCenteringParams(itemIndex = 0),
+                state = listState
+            ) {
+
+                for (satStatue in satStatues) {
+                    item {
+                        Text(text = satStatue)
+                    }
+                }
+            }
```

Comment:
Inside the `ScalingLazyColumn` below the location info, let's add more `item {}`, one for each satellite in view from GnssStatus API:

<img width="178" alt="image" src="https://user-images.githubusercontent.com/928045/198314999-947c92f2-8cc0-4f14-967c-a726c2dca645.png">

Let's try to fit ID, GNSS, CF, C/N0, and Flags columns.

Reply:
You should be able to copy/paste code from here for this section:
https://github.com/barbeau/gpstest/blob/master/GPSTest/src/main/java/com/android/gpstest/ui/status/StatusScreen.kt#L164

Or ideally, move it to the library module, although I saw your comment on having problems getting that working with the app.

Reply:
> You should be able to copy/paste code from here for this section: https://github.com/barbeau/gpstest/blob/master/GPSTest/src/main/java/com/android/gpstest/ui/status/StatusScreen.kt#L164
> 
> Or ideally, move it to the library module, although I saw your comment on having problems getting that working with the app.

Thanks, I'll do this after the code migrations from `GPSTest` to `library` work.